keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/NetworkLoadBalancerModel.kt

```diff
@@ -0,0 +1,18 @@
+package com.netflix.spinnaker.keel.clouddriver.model
+
+import com.netflix.spinnaker.keel.api.Moniker
+
+data class NetworkLoadBalancerModel(
+  override val moniker: Moniker?,
+  override val loadBalancerName: String,
+  override val loadBalancerType: String = "network",
+  override val vpcId: String,
+  override val subnets: Set<String>,
+  override val scheme: String?,
+  override val availabilityZones: Set<String>,
+  val targetGroups: List<TargetGroup>
+) : AmazonLoadBalancer {
+  data class TargetGroup(
+    val targetGroupName: String
+  )
+}
```

Comment:
Basic NLB model. We just want to check that the thing exists and has the target group we want

Reply:
It's probably better to add here a comment that we don't support it yet? So we / anyone else looking at the code will be confused 🤷